src/Framework/AspNetCoreAnalyzers/test/RouteEmbeddedLanguage/RoutePatternAnalyzerTests.cs

```diff
@@ -4,13 +4,38 @@
 using System.Globalization;
 using Microsoft.AspNetCore.Analyzer.Testing;
 using Microsoft.AspNetCore.Analyzers.RenderTreeBuilder;
+using Microsoft.AspNetCore.Analyzers.RouteEmbeddedLanguage.Infrastructure;
 
 namespace Microsoft.AspNetCore.Analyzers.RouteEmbeddedLanguage;
 
 public partial class RoutePatternAnalyzerTests
 {
     private TestDiagnosticAnalyzerRunner Runner { get; } = new(new RoutePatternAnalyzer());
 
+    [Fact]
+    public async Task CommentOnString_ReportResults()
+    {
+        var source = TestSource.Read(
+@"
+class Program
+{
+    static void Main()
+    {
+        // language=Route
```

Comment:
Do we have a test that checks that a `language=Route,Http` comment will allow processing the length-specifc constraints and the `language=Route, Component` won't?